docs/Quick-Start.md

```diff
@@ -4,74 +4,706 @@ sort: 13
+docker-compose up
 ```
 
+</div>
+
 * [Cluster setup](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#cluster-setup)
 
-## Writing data
 
-Data can be written to VictoriaMetrics in the following ways:
+## Data model
+
+### What is a metric
+
+Simply put, `metric` - is a measure or observation of something. The measurement can be used to describe the process, 
+compare it to other process, perform some calculations with it or even define events to trigger on reaching 
+user-defined thresholds.
+
+The most common use-cases for metrics are:
+- check how the system behaves at the particular time period;
+- correlate behaviour changes to other measurements;
+- observe or forecast trends;
+- trigger events (alerts) if metric exceeds threshold.
+
+Collecting and analyzing metrics provides advantages which are difficult to overestimate.
+
+### Structure of a metric
+
+Let's start with example. To track how many requests our application serves,
+we'll define a metric with name `requests_total`.
+
+You can be more specific here by saying `requests_success_total` (for only successful requests)
+or `request_errors_total` (for requests which failed). Choosing a metric name is very important and supposed 
+to clarify what is actually measured to every person who reads it, just like variable names in programming.
+
+Every metric can contain additional meta information in the form of label-value pairs:
+```
+requests_total{path="/", code="200"} 
+requests_total{path="/", code="403"} 
+```
+
+The meta information (set of `labels` in curly braces) gives us a context for which `path` and with what `code` 
+the `request` was served. Label-value pairs are always of a `string` type. VictoriaMetrics data model 
+is schemaless, which means there is no need to define metric names or their labels in advance. 
+User is free to add or change ingested metrics anytime.
+
+Actually, metric's name is also a label with a special name `__name__`. So the following two series are identical:
+```
+requests_total{path="/", code="200"} 
+{__name__="requests_total", path="/", code="200"} 
+```
+
+A combination of a metric name and its labels defines a `time series`. 
+For example, `requests_total{path="/", code="200"}` and `requests_total{path="/", code="403"}` 
+are two different time series.
+
+The number of all unique labels combinations for one metric defines its `cardinality`. 
+For example, if `requests_total` has 3 unique `path` values and 5 unique `code` values, 
+then its cardinality will be `3*5=15` of unique time series.
+
+Every time series consists of `datapoints` (also called `samples`). 
+A `datapoint` is value-timestamp pair associated with the specific series:
+```
+requests_total{path="/", code="200"} <float64 value> <unixtimestamp>
+```
+
+In VictoriaMetrics data model, datapoint's value is of type `float64`. 
+And timestamp is unixtime with milliseconds precision. Each series can contain infinite number of datapoints.
```

Comment:
Should we mention series `step` there?

Reply:
With what reference do you think it should be mentioned?